app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,27 @@
+<h2><%= t "index.sign_up" %></h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
```

Comment:
`resource` là gì em

Reply:
resource được tạo ra khi khai báo devise for Model trong file routes.rb. Nó có thể là 1 hoặc nhiều instance tùy việc devise for cho 1 hay nhiều model. Nên phía authorize của Devise sẽ dựa trên resource, chứ không phải khai báo instance model ở đây.